### PR TITLE
Make the profile custom and unique

### PR DESCRIPTION
**Pull request by evaporei**
The commits are self explanatory.

I hope you don't hate me too much after this 😅

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,31 @@
-<img height="256" align="left" src="https://user-images.githubusercontent.com/39812919/122685073-b5075180-d1df-11eb-8172-9ee6ada53190.gif" />
+## 👩 Hi!
```
Comment (antisaling) [inline README.md:1]: Should be starting with a single `#`, as it's a title.

File: README.md [hunk 2]
```diff
@@ -1,13 +1,31 @@
...
 
-#### about me
+Eu sou Isabella Basso
```
Comment (antisaling) [inline README.md:3]: In Portuguese, we usually use the definite article when referring to people's names, i.e.
```suggestion
Eu sou a Isabella Basso
```

File: README.md [hunk 3]
```diff
@@ -1,13 +1,31 @@
...
+ 
+Redpill 💊 + Desenvolvedora
```
Comment (antisaling) [inline README.md:5]: This seems kinda out of place, I'd do
```suggestion
Eu sou a Isabella Basso - Redpill 💊 + Desenvolvedora
```
or something like this.

File: README.md [hunk 4]
```diff
@@ -1,13 +1,31 @@
...
 
-- learning C + django
-- software engineer @ Red Hat Plumbers team
-- undergrad student @ [universidade de são paulo](https://www5.usp.br)
-- proud member of [lkcamp.dev](LKCamp)
+Cursando Ciências Moleculares 🧪 - USP.
```
Comment (antisaling) [inline README.md:7]: ditto

File: README.md [hunk 5]
```diff
@@ -1,13 +1,31 @@
...
 
-#### what i do
+Meu foco atualmente está em conseguir um blahaj 🦈 e em segundo plano em fazer bugs baixo nível com C (odeio Rust btw 🦀🔫)
```
Comment (antisaling) [inline README.md:9]: I'm sure this lacks punctuation

File: README.md [hunk 6]
```diff
@@ -1,13 +1,31 @@
...
 
-- now: C static analysis tool + linux kernel unit testing
-- previous: whatsapp bot + AR for the web
+<div>
+  <img height="180em" src="https://github-readme-stats.vercel.app/api?username=isinyaaa&show_icons=true&theme=tokyonight"/>
+  <img height="180em" src="https://github-readme-stats.vercel.app/api/top-langs/?username=isinyaaa&layout=compact&theme=tokyonight"/>
```
Comment (antisaling) [inline README.md:13]: Please hide markup languages/tex, as this bloats the description

File: README.md [hunk 7]
```diff
@@ -1,13 +1,31 @@
...
+</div>
+
+### ❤️ Let's get connected:
+
+[![Discord](https://img.shields.io/badge/Discord-7289DA?style=for-the-badge&logo=discord&logoColor=white)](https://discord.gg/im-not-gonna-put-your-real-discord-lol-im-not-that-evil)
```
Comment (antisaling) [inline README.md:18]: You can use my Discord in here :)

File: README.md [hunk 8]
```diff
@@ -1,13 +1,31 @@
...
+
+### 🛠️ Technologies e Tools I use:
+
+<div>
+<img align="center" alt="Isabella-c" height="30" width="40" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/c/c-original.svg"/>
+<img align="center" alt="Isabella-python" height="30" width="40" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/python/python-original.svg"/>
+<img align="center" alt="Isabella-django" height="30" width="40" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/django/django-original.svg"/>
+<img align="center" alt="Isabella-linux" height="30" width="40" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/linux/linux-original.svg"/>
+
+
```
Comment (antisaling) [inline README.md:30]: Unnecessary whitespace here.

File: README.md [hunk 9]
```diff
@@ -1,13 +1,31 @@
...
+[![Linkdin](https://img.shields.io/badge/LinkedIn-0077B5?style=for-the-badge&logo=linkedin&logoColor=white)](https://www.linkedin.com/in/isabellabasso/)
+[![Twitter](https://img.shields.io/badge/Twitter-1DA1F2?style=for-the-badge&logo=twitter&logoColor=white)](https://twitter.com/isinyaaa)
+
+### 🛠️ Technologies e Tools I use:
+
+<div>
+<img align="center" alt="Isabella-c" height="30" width="40" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/c/c-original.svg"/>
+<img align="center" alt="Isabella-python" height="30" width="40" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/python/python-original.svg"/>
+<img align="center" alt="Isabella-django" height="30" width="40" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/django/django-original.svg"/>
+<img align="center" alt="Isabella-linux" height="30" width="40" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/linux/linux-original.svg"/>
```
Comment (antisaling) [inline README.md:28]: Isn't this list missing on some key technologies? I.e. firebase, latex... Have a look at [isinyaaa.github.io/](https://isinyaaa.github.io/CV.pdf)

File: README.md [hunk 10]
```diff
@@ -1,13 +1,31 @@
...
+Meu foco atualmente está em conseguir um blahaj 🦈 e em segundo plano em fazer bugs baixo nível com C (odeio Rust btw 🦀🔫)
 
-- now: C static analysis tool + linux kernel unit testing
-- previous: whatsapp bot + AR for the web
+<div>
+  <img height="180em" src="https://github-readme-stats.vercel.app/api?username=isinyaaa&show_icons=true&theme=tokyonight"/>
+  <img height="180em" src="https://github-readme-stats.vercel.app/api/top-langs/?username=isinyaaa&layout=compact&theme=tokyonight"/>
+</div>
+
+### ❤️ Let's get connected:
```
Comment (antisaling) [inline README.md:16]: Fixup this to match my above comment. I.e. should be
```suggestion
## ❤️ Let's get connected:
```

Comment (antisaling) [reply README.md:16]: Are you sure this should be done in English? I think it'd be best to use a single language throughout...

**Comment by antisaling** [comment]
Oh... Let's play 😈

**Review by antisaling — CHANGES_REQUESTED** [review]
Hi, @otaviopace. Firstly, thanks for the contribution!

I tried taking your review as seriously as I would take any, so I'm sorry if it was harsh, as this seems a very personal-taste kind of change.

One point I'd like to bring up was the use of language: I think this section would be better suited for German or an Arabic language. Would you be willing to try this??

Anyway, looking forward to your updates :)

**Review by antisaling — CHANGES_REQUESTED** [review]
So, commit ccf21a7 is using very intimate language, and it's also not imperative throughout. So I'd do something like
```raw
As the profile has too much unique information, scrap it and start anew.

Using a BOC GIF isn't at all appropriate as it shows a matter of personal taste
and one should follow the clear standard well put in place by the content
creators of our (amazing) developer community.

As it's the standard, any developer should just follow it like everybody else
in order to become blended in our community.
```

Now put this in your PR description.

```
Unfortunately, I had to start over from scratch, the following commits
will make your Github profile README.md finally appropriate.
```

**Review by antisaling — CHANGES_REQUESTED** [review]
I'd squash 60b4094 with the previous commit (I mean commits 1/5 and 2/5), as they're very much related.

Also change the commit message to use imperative language.

**Review by antisaling — CHANGES_REQUESTED** [review]
Typos in commit message 3/5 (3a3f95c)

```raw
Having a GitStats is **crucial** to show to companies
```

**Review by antisaling — CHANGES_REQUESTED** [review]
Change 5/5 (4cec0680) commit message
```raw
readme: Add Technologies **&** Tools I use section
```

Also, as I said previously, please change the language to imperative throughout the commit.

**Comment by evaporei** [comment]
Hey @isinyaaa I really appreciate the thought and care you've put into this review, you really took it seriously 😅 

Unfortunately this is a joke, and a lot of the mistakes were done because the original material/tutorial actually had them, I left on purpose so it looks more corny.

I even thought at some point to leave the commits in portuguese as well, but yeah, I wanted to do other stuff over the weekend haha 🛼 

See ya someday 👋 🐢 🐵

**Comment by antisaling** [comment]
@otaviopace would you check your telegram please?

**Comment by evaporei** [comment]
lol
